models/User.js

```diff
@@ -31,7 +31,11 @@ const UserSchema = new mongoose.Schema({
     type: String,
     required: true
   },
-  activities: [String]
+  activities: [String],
+  alertsViewed: [{
+    name: String,
```

Comment:
we could just store an array of alert ID's, no?  Do we need the name and description?

Reply:
Yeah you're right, I'll do it like that